src/video_stream_py/video_stream_py/video_publisher_imageio.py

```diff
@@ -0,0 +1,47 @@
+import rclpy
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+import imageio.v3 as iio
```

Comment:
I'm guessing this `iio` import is the only difference between this module and `video_publisher_py.py` module?

Reply:
video_publisher_imageio.py uses the imageio library as opposed to CV2 because we were hoping to use a lighter library (CV2 has so much unnecessary stuff and using a lighter library was a simpler alternative to specifically installing certain cv2 dependencies) Main difference is in how frames are being sent over